### ISSUE TITLE
Phase out weak ciphers?

### ISSUE THREAD

**Issue by cpuguy83** [docker/go-connections#101]
We have this comment: https://github.com/docker/go-connections/blob/58542c764a1173ea3dac965d89146c931a2946f7/tlsconfig/config.go#L43

Perhaps it is time to go ahead and make good on that comment?

**Comment by neersighted** [comment]
PTAL @corhere as well; I have no strong opinions -- if no others appear, maybe we discuss this on Thursday?

**Comment by thaJeztah** [comment]
Yeah, we still need to dig into the TLS1.3 issue as well; updating moby to the latest code in this repo breaks things (see https://github.com/moby/moby/pull/41084 and the original PR linked)

**Comment by cpuguy83** [comment]
Seems strange that we even have tlsconfig in this repo.

**Comment by thaJeztah** [comment]
Yeah, ISTR the original reason was to have a canonical place where these were defined (as at the time it was not always trivial to decide on the ciphers, and to make sure all repositories followed the same)
